quiz/src/test/java/rx/quiz/combining/IntermediateCombineLatestKoanTest.java

```diff
@@ -0,0 +1,46 @@
+package rx.quiz.combining;
+
+import io.reactivex.Observable;
+import org.jetbrains.annotations.NotNull;
+
+public class IntermediateCombineLatestKoanTest extends IntermediateCombineLatestKoan {
+
+    @NotNull
+    @Override
+    public Observable<Sound> parkingAssistant(@NotNull ParkingApi api) {
+        return Observable.combineLatest(
+                api.isSwitchedOn(),
+                api.isRearSpeedOn(),
+                api.frontLeft(),
+                api.frontRight(),
+                api.rearLeft(),
+                api.rearRight(),
+                (isSwitchedOn, isRearSpeedOn, frontLeft, frontRight, rearLeft, rearRight) -> {
```

Comment:
Very good solution with Combine latest. I have also seen variants with switchMap which did not look so elegant.